src/components/VaccineProgressBar/VaccineProgressBar.tsx

```diff
@@ -2,15 +2,14 @@ import React from 'react';
 import { ParentSize } from '@vx/responsive';
 import { v4 as uuidv4 } from 'uuid';
 import { ProgressBarContainer, StyledSvg } from './VaccineProgressBar.style';
-import { COLOR_MAP, VACCINATIONS_COLOR_MAP, vaccineColor } from 'common/colors';
+import { VACCINATIONS_COLOR_MAP, vaccineColor } from 'common/colors';
 import { formatPercent } from 'common/utils';
 
 export interface ProgressBarProps {
   // TODO(michael): Remove once we migrate the location page to the new version.
   oldVersion?: boolean;
   locationName: string;
-  vaccinationsInitiated: number;
-  vaccinationsCompleted: number;
+  vaccinationsRatio: number;
```

Comment:
We're modifying the progress bar to just display one line, as it would be unclear which doses were being displayed with multiple lines. 

Arguably, we could instead create a new component for this, but since we only use this one-bar version I'm not sure it's worth it. This does require a few changes throughout the codebase, though, that maybe could be avoided, so I could be convinced to rework this

Reply:
I think it's fine!